server/utils/drupal/drupal-forums.js

```diff
@@ -1,13 +1,20 @@
-import { stripHtml                } from 'string-strip-html' ;
-import { camelCase                } from 'change-case/keys'  ;
+import { stripHtml }                from 'string-strip-html' ;
+import { camelCase }                from 'change-case/keys'  ;
 import { validate  as validateUuid} from 'uuid'              ;
+import   clone                      from 'lodash.clonedeep'  ;
+
 
 export const useDrupalForums = async (ctx) => {
+
     return getForums(ctx)
 }
 
 export async function addForumIdentifierToContext(ctx){
 
+    if(Number(ctx.forumAlias)) { 
```

Comment:
cool. Always thougth that NaN return true
<img width="322" alt="image" src="https://github.com/user-attachments/assets/c0714d0e-af46-4be0-9ce7-874b7bdcbaf4" />